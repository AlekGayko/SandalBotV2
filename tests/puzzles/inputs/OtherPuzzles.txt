10000 8/k7/3p4/p2P1p2/P2P1P2/8/8/K7 w - - 0 1 
5000 2k3r1/2p1q1r1/p3b2p/1p3B2/8/2N5/PP1BQPPP/2R2RK1 b - - 0 1 
1500 6k1/2QN1p1p/p3p1p1/3pP1N1/1r3n2/8/5PPP/5K2 b - - 3 31 