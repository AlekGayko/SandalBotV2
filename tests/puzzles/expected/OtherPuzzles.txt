a1b1 Sebastian Lague Endgame (Transposition Table)
g7g2 2400 Elo Sacrifice M4
b4b1 188 Elo Back Rank Checkmate